File: BarberBooking.API/Dto/DtoVo/DtoAddress.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace BarberBooking.API.Dto.DtoVo
{
    public class DtoAddress
    {
        public string City {get; set;}
        public string Street {get; set;}
        public string HouseNumber{get; set;}
        public string Apartment {get; set;}
    }
}

File: barber_booking_app/lib/models/VoModels/dto_address.dart
class DtoAddress {
  String? City;
  String? Street; 
  String? HouseNumber;
  String? Apartment; 

  DtoAddress({this.City, this.Street, this.HouseNumber, this.Apartment});

  factory  DtoAddress.fromJson(Map<String, dynamic> json){
      return  DtoAddress(
          City: json['city'],
          Street: json['street'],
          HouseNumber: json['houseNumber'],
          Apartment: json['apartment'],
      );
    }
}